exo-go/src/app_runner/index.go

```diff
@@ -63,6 +70,31 @@ func (a *AppRunner) getEnv() []string {
 	return formattedEnvVars
 }
 
+func (a *AppRunner) runImages(imageNames []string, imageOnlineTexts map[string]string, identifier string) error {
+	process, err := dockerCompose.RunImages(imageNames, a.getEnv(), a.DockerConfigLocation, a.write)
+	if err != nil {
+		return errors.Wrap(err, fmt.Sprintf("Failed to run %s\nOutput: %s\nError: %s\n", identifier, process.Output, err))
+	}
+	var wg sync.WaitGroup
+	var onlineTextRegex *regexp.Regexp
+	for role, onlineText := range imageOnlineTexts {
+		wg.Add(1)
+		onlineTextRegex, err = regexp.Compile(fmt.Sprintf("%s.*%s", role, onlineText))
+		if err != nil {
+			return err
+		}
+		go func(role string, onlineTextRegex *regexp.Regexp) {
+			a.waitForOnlineText(process, role, onlineTextRegex)
+			wg.Done()
+		}(role, onlineTextRegex)
+	}
+	wg.Wait()
+	if err == nil {
```

Comment:
I know you were just copying code, but I believe this check can go away as err will never be non-nil at this point.